google/cloud/forseti/common/data_access/violation_map.py

```diff
@@ -47,6 +47,7 @@
     'LOCATION_VIOLATION': 'location_violations',
     'LOG_SINK_VIOLATION': 'log_sink_violations',
     'RETENTION_VIOLATION': 'retention_violations',
+    'ROLE_VIOLATION': 'custom_role_violations',
```

Comment:
Change this to either
`'CUSTOM_ROLE_VIOLATION': 'custom_role_violations',`
or
`'ROLE_VIOLATION': 'role_violations',`

So that is matches the pattern of other violations.